drivers/hmis/app/models/hmis/filter/organization_filter.rb

```diff
@@ -0,0 +1,20 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class Hmis::Filter::OrganizationFilter < Hmis::Filter::BaseFilter
+  def filter_scope(scope)
+    scope = ensure_scope(scope)
+    scope.
+      yield_self(&method(:with_search_term)).
+      yield_self(&method(:clean_scope))
```

Comment:
We don't really need this, since the search term scope doesn't do a `join`. It was added in some places to deal with that, because the join would lead to duplicate search results. This is a pretty small table, so the performance impact is probably not noticable, but I would err on the side of removing it since it's not needed.